ocdata/core/surface.py

```diff
@@ -0,0 +1,275 @@
+import math
+import os
+import pickle
+from collections import defaultdict
+
+import numpy as np
+from ase import neighborlist
+from ase.constraints import FixAtoms
+from pymatgen.analysis.local_env import VoronoiNN
+from pymatgen.core import Composition
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+
+from ocdata.configs.constants import MIN_XY
```

Comment:
Similar to MAX_MILLER, let's not hard code this constant.

Reply:
Updated here: https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset/commit/5d1ed1fdcbe8c8b9765d8930288689bc7883fcdb